src/ocean_emulators/utils/data.py

```diff
@@ -115,6 +132,16 @@ def normalize_with(
         norm = norm.to(data.dtype)
         return norm
 
+    @classmethod
+    def from_dataset(cls, dataset: xr.Dataset) -> Self:
+        """Create a DataSource from the data, computing means and stds."""
+        assert "lev" not in dataset.dims, "We currently only support non-compact data"
+
+        means = dataset.map(lambda x: x.mean())
+        stds = dataset.map(lambda x: x.std())
```

Comment:
Any reason why we can't just do the following?
```suggestion
        means = dataset.mean()
        stds = dataset.std()
```

Reply:
Nope, thanks!